### ISSUE TITLE
Migrate from TravisCI to GitHub Actions

### ISSUE THREAD

**Issue by tylerwoonton** [ukfast/sdk-php#192]
### What is the feature?

I'd like to suggest moving to GitHub Actions for our CI workflow and moving away from our current TravisCI implementation.

I'd definitely like to get other opinions on this.

### What benefits will this change bring?

This will allow us to keep all our workflow for the SDK on GitHub.

We could leverage a lot of the prebuilt workflows, maintaining functionality from our existing pipelines and adding additional functionality too.

### Additional context

From what I've read about GitHub Actions and my limited exposure to them, it shouldn't be too hard to migrate our Travis config over.

**Comment by Gman98ish** [comment]
![image](https://user-images.githubusercontent.com/7416827/72879663-432e8e00-3cf5-11ea-8b02-59305e53fc21.png)

I have no strong feelings one way or another

**Comment by gavtaylor** [comment]
> I'd definitely like to get other opinions on this.

no objections from me, I only choose Travis because I've used it before, happy to move to Actions if its a better option.

got some time to pop a PR in for it? :)

**Comment by tylerwoonton** [comment]
> got some time to pop a PR in for it? :)

I'll stick a PR in for it tonight, just need to match our config.

**Comment by tylerwoonton** [comment]
@gavtaylor What do we think about GitHub Actions then? We've got it running in the health check now and it's pretty nice imo. 

We can do a lot more than this current config does but this replicates what we do on Travis without the need for external services. I'd be keen to see if there were any objections to moving this over.

It was reviewed and approved by @MisterCoder. I've just tweaked the config slightly to match the one used in the health-check package.
